### PR TITLE
chore: set `p0tion` /`trusted-setups`/`ppot` to archived/inactive

### PR DESCRIPTION
**Pull request by sripwoud**

### CODE REVIEW

File: data/projects/p0tion.ts [hunk 1]
```diff
@@ -9,15 +9,18 @@ const content: ProjectContent = {
...
   content,
   image: "p0tion.png",
   name: "p0tion",
   links: {
     website: "https://ceremony.pse.dev/",
     github: "https://github.com/privacy-scaling-explorations/p0tion",
   },
+  // extraLinks: {
+  //   learn: [{ label: "Trusted Setup/P0tion LTS Announcement", url: "TODO" }],
+  // },
```
Comment (sripwoud) [inline data/projects/p0tion.ts:23]: @NicoSerranoP is preparing something we will publish to communicate this.
So to be update later

File: data/projects/trusted-setups.ts [hunk 2]
```diff
@@ -9,14 +9,17 @@ const content: ProjectContent = {
...
+  projectStatus: ProjectStatus.INACTIVE,
   content,
   image: "trusted-setups.svg",
   name: "Trusted Setups",
   links: {
     github: "https://github.com/zkparty",
   },
+  extraLinks: {
+    learn: [{ label: "Trusted Setup/P0tion LTS Announcement", url: "TODO" }],
+  },
```
Comment (sripwoud) [inline data/projects/trusted-setups.ts:22]: @NicoSerranoP is preparing something we will publish to communicate this.
So to be update later

**Comment by NicoSerranoP** [comment]
Hi @sripwoud I created this PR last week 😊 . It has the same changes plus a status description talking about why we archived the projects. Could you check it out to see if we merge that one instead of this PR? 

https://github.com/privacy-scaling-explorations/pse.dev/pull/241 #241

**Comment by sripwoud** [comment]
duplicate of #241

**Pull request closed without merging**